odtools/odrecinspect/include/RecInspect.h

```diff
@@ -69,6 +69,7 @@ namespace odrecinspect {
         private:
             std::map<int32_t, uint64_t> m_numberOfContainersPerType;
             std::map<int32_t, odcore::data::Container> m_latestContainersPerType;
+            std::map<int32_t, uint32_t> m_numberOfContainersInIncorrectTemporalOrderPerType;
```

Comment:
OK. It looks like a typo for the first glance. But your explanation makes sense:)